### ISSUE TITLE
Laravel 5.0 Not Supported

### ISSUE THREAD

**Issue by jolora** [joshrps/laravel-shopify-API-wrapper#1]
This looks great. Thanks! But I'm guessing it's not compatible with Laravel 5.0? This is what I get when I try to use composer to add the API wrapper to a blank install of Laravel 5.0:

```
$ composer update rocket-code/shopify
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove laravel/framework v5.0.2
```

**Comment by joshrps** [comment]
Haven't played around with Laravel 5 yet, I'll get on it!

**Comment by joshrps** [comment]
Dependencies fixed in 1.0.6!
